python/test/gluon/test_frontend.py

```diff
@@ -617,8 +626,9 @@ def test_warpgroup_mma():
 @gluon.jit
 def warpgroup_mma_wait_kernel():
     layout: ttgl.constexpr = ttgl.NVMMADistributedLayout(version=[3, 0], warps_per_cta=[4, 1], instr_shape=[16, 32, 16])
-    acc = ttgl.full([128, 128], 0, dtype=ttgl.float16, layout=layout)
+    acc = hopper.warpgroup_mma_init(ttgl.full([128, 128], 0, dtype=ttgl.float16, layout=layout))
     acc = hopper.warpgroup_mma_wait(num_outstanding=1, deps=[acc])
```

Comment:
I feel like we should hide the `load` method and make `warpgroup_mma_wait` unwrap the future and return the tensor. Since I presume the purpose of the API is to force users to route the `warpgroup_mma` result through a `warpgroup_mma_wait` before accessing the value.